composer.json

```diff
@@ -29,7 +29,8 @@
         "riverline/multipart-parser": "^2.0.6",
         "psr/http-server-handler": "^1.0",
         "nyholm/psr7": "^1.4.1",
-        "psr/container": "^1.0|^2.0"
+        "psr/container": "^1.0|^2.0",
+        "crwlr/query-string": "^1.0"
```

Comment:
Do we want to alphabetically sort the package requirements?

Reply:
I think we should also go with ^1.0.1` here.

Reply:
For the latter: Yes, I think the changes from 1.0.1 are pretty important, as I actually copied over the test cases from https://github.com/brefphp/bref/pull/1383 and fixed/changed a few things for that. 
But I also don't know if there's any scenario where composer would now install 1.0.0, as there are no dependencies that could conflict and nothing was changed in composer.json for 1.0.1 🤔 but maybe better be safe than sorry 👍🏻

Btw. just to mention it: maybe changes from 1.0.1 could be considered breaking. I also thought about tagging 2.0.0 but went for 1.0.1 then. But I'll definitely be very careful with changes and stick to semver very strictly when bref now uses it! 🫡

Reply:
Yes, best to insist on at least 1.0.1, if the code at 1.0.0 does not work.